sql/python/sqlflow/template/db.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class DataBase(object):
+    """A database interface """
+
+    def __init__(self, database, user, password, host, port):
+        self._database = database
+        self._user = user
+        self._password = password
+        self._host = host
+        self._port = port
+
+        self._db = self._connect()
+
+    def close(self):
+        self._db.close()
+
+    def execute(self, query):
+        """ Execute the query
+        Args:
+            query(str): the query to execute
+
+        Returns:
+            cursor(cursor):
+        """
+        cursor = self._db.cursor()
+        cursor.execute(query)
+        return cursor
+
+    def execute_many(self, query, values):
+        """ Execute the query with values
+        Args:
+            query(str): the query to execute
+            values(list(tuple)): the values
+
+        Returns:
+            cursor(cursor)
+        """
+        cursor = self._db.cursor()
+        cursor.executemany(query, values)
+        return cursor
+
+    def create_table(self, table_name, table_schema):
```

Comment:
Which part of the codegen needs this method?

Reply:
Should we create a table first before saving the prediction result?

Reply:
The current implementation does this in Go.

https://github.com/sql-machine-learning/sqlflow/blob/652aaaa4096f2bc2cb580ed7aa39ad81148c959d/sql/executor.go#L304-L306